src/export/digitalPlanning/model.ts

```diff
@@ -420,14 +438,15 @@ export class DigitalPlanning {
 
   private getApplicationFee(): Payload["data"]["application"]["fee"] {
     const baseFee = {
-      calculated: this.passport.data?.["application.fee.calculated"],
-      payable: this.passport.data?.["application.fee.payable"],
+      calculated:
+        this.passport.generic<number>("application.fee.calculated") || 0,
```

Comment:
`genric` is very much a temporary placeholder name, but it allowed me to make progress and test this without changing the current `get()` functionality in an already large PR.